qdrant-landing/content/documentation/quick-start.md

```diff
@@ -54,151 +35,67 @@ client = QdrantClient(":memory:")
 client = QdrantClient(path="path/to/db")  # Persists changes to disk
 ```
 
-Local mode is useful for development, prototyping and testing.
-
-* You can use it to run tests in your CI/CD pipeline.
-* Run it in Colab or Jupyter Notebook, no extra dependencies required. See a [Colab Example](https://colab.research.google.com/drive/1Bz8RSVHwnNDaNtDwotfPj0w7AYzsdXZ-?usp=sharing)
-* When you need to scale, simply switch to server mode.
+## Option 2: Docker Mode
 
+**Prerequisite:** First make sure Docker is installed and running on your system.
 
-## API
+### Download the image
 
-All interaction with Qdrant takes place via the REST API.
-
-This example covers the most basic use-case - collection creation and basic vector search.
-For additional information please refer to the [API documentation](https://qdrant.github.io/qdrant/redoc/index.html).
-
-### gRPC
+```bash
+docker pull qdrant/qdrant
+```
 
-In addition to REST API, Qdrant also supports gRPC interface.
-To enable gPRC interface, specify the following lines in the [configuration file](https://github.com/qdrant/qdrant/blob/master/config/config.yaml):
+### Run the service
 
-```yaml
-service:
-  grpc_port: 6334
+```bash
+docker run -p 6333:6333 \
+    -v $(pwd)/qdrant_storage:/qdrant/storage \
+    qdrant/qdrant
 ```
 
-gRPC interface will be available on the specified port.
-The gRPC methods follow the same principles as REST. For each REST endpoint, there is a corresponding gRPC method.
-Documentation on all available gRPC methods and structures is available here - [gRPC Documentation](https://github.com/qdrant/qdrant/blob/master/docs/grpc/docs.md).
+Under the default configuration all data will be stored in the `./qdrant_storage` directory.
```

Comment:
```suggestion
Under the default configuration all data will be stored in the `./qdrant_storage` directory. This will also be the only directory that both the Container and the host machine can both see. 
```